docs/bare_metal/using_bare_metal.md

```diff
@@ -1,3 +1,72 @@
 # Using the bare metal profile
 
-Content still in PR
+This guide shows how to create a bare metal profile application:
+1. By default, when you build an application binary, the build tool uses the full profile. To use the bare metal profile, you need to set up your application to override this default behaviour.
+1. Bare metal has a minimal set of APIs. You can add additional ones [from the list of supported API](bare_metal.md#features).
+
+## Creating a bare metal application
+
+1. Create a new Mbed application and enter its directory:
```

Comment:
I would introduce the bare metal application that we are building (blinky using EventQueue) by describing what the application does and showing its `main.cpp`.

Reply:
`main.cpp` is quoted below (if the doc compiler inlines the link). Do you suggest showing it first?